### ISSUE TITLE
Problem with DOM insertion to a set of matched elements

### ISSUE THREAD

**Issue by JamesBoon** [cheeriojs/cheerio#662]
If you try to append/prepend or insert after/before any element to a set of matched elements it will only do this once.
Take a look at the following example:
```javascript
var cheerio = require('cheerio'),
    $ = cheerio.load('<ul><li>A</li><li>B</li></ul>');
$('ul').children().after('<li>---</li>');
console.log($.html());
```
This *should* output:
```html
<ul><li>A</li><li>---</li><li>B</li><li>---</li></ul>
```
But it creates:
```html
<ul><li>A</li><li>B</li><li>---</li></ul>
```
This was tested with cheerio v0.18.0 and compared to jQuery v2.1.3

**Comment by JamesBoon** [comment]
This will create the expected output:
```javascript
$('ul').children().after(function() {
    return '<li>---</li>'
});
```
This seems to be a bug...

**Comment by jugglinmike** [comment]
This bug was resolved by 466e34aa76da11e6a2dd99d1bc634f1a4e2a1ee2 (via gh-574) and [released in version 0.19.0](https://github.com/cheeriojs/cheerio/commit/9e3746d391c47a09ad5b130d770c747a0d673869). Thanks for the report!
